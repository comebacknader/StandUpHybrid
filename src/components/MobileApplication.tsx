import { StatusBar } from 'expo-status-bar';
import React from 'react';
import { Text, View } from 'react-native';

import { Layout } from '../styles';

export default function MobileApplication() {
  return (
    <View style={[Layout.isFlex, Layout.justifyContentCenter, Layout.alignItemsCenter]}>
      <Text>MobileApplication</Text>
      <StatusBar style="auto" />
    </View>
  )
}