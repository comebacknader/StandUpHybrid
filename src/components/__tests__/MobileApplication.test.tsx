import React from 'react';
import renderer from 'react-test-renderer';
import MobileApplication from '../MobileApplication';

it('MobileApplication renders correctly', () => {
  const tree = renderer.create(<MobileApplication />).toJSON();
  expect(tree).toMatchSnapshot();
});