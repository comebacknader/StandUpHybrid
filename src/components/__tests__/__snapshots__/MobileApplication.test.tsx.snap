// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileApplication renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "justifyContent": "center",
      },
      Object {
        "alignItems": "center",
      },
    ]
  }
>
  <Text>
    MobileApplication
  </Text>
</View>
`;
